sjsonnet/src/sjsonnet/BaseCharRenderer.scala

```diff
@@ -154,7 +157,7 @@ class BaseCharRenderer[T <: upickle.core.CharOps.Output]
 
   private def visitNonNullString(s: CharSequence, index: Int) = {
     flushBuffer()
-    upickle.core.RenderUtils.escapeChar(null, elemBuilder, s, escapeUnicode)
+    upickle.core.RenderUtils.escapeChar(null, elemBuilder, s, escapeUnicode = escapeUnicode, wrapQuotes = true)
```

Comment:
note: pay attention here `wrapQuotes = true` comes from lib update. I checked and I think true is a valid value